docs/source/jupyter/ontology-interface.ipynb

```diff
@@ -327,52 +814,81 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 22,
    "metadata": {
     "tags": []
    },
    "outputs": [
     {
-     "output_type": "stream",
      "name": "stdout",
+     "output_type": "stream",
      "text": [
-      "\nYou can get the argument name of an attribute. The argument name is used when instantiating CUDS objects\nage\n\nYou can get the datatype of attributes\nhttp://www.w3.org/2001/XMLSchema#integer\n\nYou can use the attribute to convert values to the datatype of the attribute\n10\n"
+      "\n",
+      "You can get the argument name of an attribute. The argument name is used as keyword argument when instantiating CUDS objects.\n",
+      "age\n",
+      "\n",
+      "You can get the datatype of attributes\n",
+      "http://www.w3.org/2001/XMLSchema#integer\n",
+      "\n",
+      "You can use the attribute to convert values to the datatype of the attribute\n",
+      "<class 'int'> 10\n",
+      "\n",
+      "And likewise to convert values to the python basic type associated with the datatype of the attribute.\n",
+      "<class 'str'> 5\n"
      ]
     }
    ],
    "source": [
     "print(\"\\nYou can get the argument name of an attribute. \"\n",
-    "      \"The argument name is used when instantiating CUDS objects\")\n",
+    "      \"The argument name is used as keyword argument when instantiating CUDS objects.\")\n",
     "print(city.age.argname)\n",
     "\n",
     "print(\"\\nYou can get the datatype of attributes\")\n",
     "print(city.age.datatype)\n",
     "\n",
     "print(\"\\nYou can use the attribute to convert values \"\n",
     "      \"to the datatype of the attribute\")\n",
-    "print(city.age.convert_to_datatype(\"10\"))"
+    "result = city.age.convert_to_datatype(\"10\")\n",
```

Comment:
Is there a documentation on `convert_to_datatype`? Seems not to be in `/api_ref.html`. For me this function was confusing.

Reply:
Added.